<launch>
    <param name="webserver_port" type="int" value="8000" />
    <param name="websocket_port" type="int" value="8001" />
    
    <node name="WebServer" pkg="webserver" type="WebServer.py" />
</launch>